.github/workflows/ci.yml

```diff
@@ -47,9 +47,21 @@ jobs:
         run: yarn lint
 
       - name: Run happo
+        if: github.event_name == 'pull_request'
```

Comment:
this workflow runs for both `push` and `pull_request` actions. I don't think we need to re-run this upon PR merge.